var/spack/repos/builtin/packages/intel-oneapi-mkl/package.py

```diff
@@ -34,13 +34,19 @@ class IntelOneapiMkl(IntelOneApiLibraryPackage):
                 sha256='818b6bd9a6c116f4578cda3151da0612ec9c3ce8b2c8a64730d625ce5b13cc0c',
                 expand=False)
 
+    variant('shared', default=True, description='Builds shared library')
     variant('ilp64', default=False,
             description='Build with ILP64 support')
+    variant('cluster', default=False,
+            description='Build with cluster support: scalapak, blacs, etc')
 
     depends_on('intel-oneapi-tbb')
+    # cluster libraries need mpi
+    depends_on('mpi', when='scalapack')
```

Comment:
I don't think this will have any effect: I think this is saying `This depends on MPI when it provides scalapack` but I don't think we actually support this type of condition. Usually people copy the conditions under which the virtual is provided; I see that on the next line so I think you can just omit this `depends_on`.

Reply:
Removed